doc/lua_api.txt

```diff
@@ -2299,7 +2299,7 @@ Elements
 * `frame duration`: Milliseconds between each frame. `0` means the frames don't advance.
 * `frame start` (Optional): The index of the frame to start on. Default `1`.
 
-### `model[<X>,<Y>;<W>,<H>;<name>;<mesh>;<textures>;<rotation X,Y>;<continuous>;<mouse control>;<frame loop range>]`
+### `model[<X>,<Y>;<W>,<H>;<name>;<mesh>;<textures>;<rotation X,Y>;<continuous>;<mouse control>;<frame loop range>;<animation speed>]`
```

Comment:
Should be noted that 5.4 doesn't support animation speed (and will ungracefully not display it at all).

Reply:
I’m assuming most models are static, so better not animate them by default with an arbitrary and potentially inadequate value.